### PR TITLE
Update CMBChina.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
https://acs.cmbchina.com/MasterCard/PAMsg.aspx
https://b2b.cmbchina.com/CmbBank_B2B/UI/Home.aspx
https://cm.cmbchina.com/
https://ebank.nj1.cmbchina.com/
https://mlife.cmbchina.com/
https://netpay.cmbchina.com/netpayment/images/NetPayApplyCard.htm
https://ssl.img.jf.cmbchina.com/images/CM/P60/A0C-706-021.jpg

懒得找了ω

**Comment by ivysrono** [comment]
@J0WI I have try my best for this Chinese' largest joint-stock commercial banks.

**Comment by J0WI** [comment]
See also:
link.cmbchina.com (e.g. on `http://link.cmbchina.com/cmbcareer2015/`)
info.cmbchina.com
pension.cmbchina.com
fi.cmbchina.com
sg.cmbchina.com
futures.cmbchina.com
big5.cmbchina.com
london.cmbchina.com
english.cmbchina.com
gold.cmbchina.com
gb.cmbchina.com
fx.cmbchina.com

File: src/chrome/content/rules/CMBChina.com.xml [hunk 1]
```diff
@@ -1,39 +1,139 @@
...
+	<target host="user.cmbchina.com" />
+	<target host="xiaozhao.wx.cmbchina.com" />
+	<target host="xyk.cmbchina.com" />
+	<target host="xykstatic.cmbchina.com" />
+
+	<!--	Complications:	-->
 	<target host="www.cmbchina.com" />
-		<exclusion pattern="^http://www\.cmbchina\.com/$" />
-		<test url="http://www.cmbchina.com/" />
+	<exclusion pattern="^http://www\.cmbchina\.com/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/CMBChina.com.xml]: You exclude only `$`, but it seems that there are other affected pages.

File: src/chrome/content/rules/CMBChina.com.xml [hunk 2]
```diff
@@ -1,39 +1,139 @@
...
+		<test url="http://www.cmbchina.com/personal/" />
 
-	<rule from="^http:" to="https:" />
+	<target host="ccclub.cmbchina.com" />
+	<exclusion pattern="^http://ccclub\.cmbchina\.com/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/CMBChina.com.xml]: You exclude only `$`, but it seems that there are other affected pages.

File: src/chrome/content/rules/CMBChina.com.xml [hunk 3]
```diff
@@ -1,39 +1,139 @@
...
+		<test url="http://live.cmbchina.com/cmbwebshare/Scripts/CmbWebShare.js" />
+		<test url="http://live.cmbchina.com/CMU/images/tp0000.gif" />
+		<test url="http://live.cmbchina.com/cmu/images/announce.gif" />
+		<test url="http://live.cmbchina.com/cmu/scripts/default.js" />
+		<test url="http://live.cmbchina.com/cmu/style/default.css" />
+		<test url="http://live.cmbchina.com/webpages/firmbankweb/zhiyin.htm" />
+		<test url="http://live.cmbchina.com/webpages/fxexchange/index.html" />
+
+	<target host="market.cmbchina.com" />
+	<exclusion pattern="^http://market\.cmbchina\.com/ccard/business/" />
```
Comment (J0WI) [inline src/chrome/content/rules/CMBChina.com.xml]: You exclude only `/ccard/business/`, but it seems that there are other affected pages.

**Comment by ivysrono** [comment]
big5.cmbchina.com is blocked by google?

**Comment by J0WI** [comment]
Thanks!
big5 is on the [safebrowsing](https://www.google.com/transparencyreport/safebrowsing/diagnostic/#url=big5.cmbchina.com) list of Google. The Tor Browser Bundle is not using it, but for now I'm okay with not adding it.

**Pull request merged**